editor/test/integration/app_manifest_test.clj

```diff
@@ -147,167 +166,217 @@
+        (is (= :open-gl (g/node-value manifest :graphics)))
+        (is (= :both (g/node-value manifest :graphics-osx)))
+        (is (= :web-gl (g/node-value manifest :graphics-web)))))
+    (testing "/app_manifest/webgpu.appmanifest"
+      (let [manifest (test-util/resource-node project "/app_manifest/webgpu.appmanifest")]
+        (is (= :legacy (g/node-value manifest :physics-2d)))
+        (is (= true (g/node-value manifest :physics-3d)))
+        (is (= false (g/node-value manifest :exclude-record)))
+        (is (= false (g/node-value manifest :exclude-profiler)))
+        (is (= false (g/node-value manifest :exclude-sound)))
+        (is (= false (g/node-value manifest :exclude-input)))
+        (is (= false (g/node-value manifest :exclude-liveupdate)))
+        (is (= false (g/node-value manifest :exclude-basis-transcoder)))
+        (is (= false (g/node-value manifest :use-android-support-lib)))
+        (is (= :open-gl (g/node-value manifest :graphics)))
+        (is (= :vulkan (g/node-value manifest :graphics-osx)))
+        (is (= :web-gpu (g/node-value manifest :graphics-web)))))
+    (testing "/app_manifest/webgpu_and_webgl.appmanifest"
+      (let [manifest (test-util/resource-node project "/app_manifest/webgpu_and_webgl.appmanifest")]
+        (is (= :legacy (g/node-value manifest :physics-2d)))
+        (is (= true (g/node-value manifest :physics-3d)))
+        (is (= false (g/node-value manifest :exclude-record)))
+        (is (= false (g/node-value manifest :exclude-profiler)))
+        (is (= false (g/node-value manifest :exclude-sound)))
+        (is (= false (g/node-value manifest :exclude-input)))
+        (is (= false (g/node-value manifest :exclude-liveupdate)))
+        (is (= false (g/node-value manifest :exclude-basis-transcoder)))
+        (is (= false (g/node-value manifest :use-android-support-lib)))
+        (is (= :open-gl (g/node-value manifest :graphics)))
+        (is (= :vulkan (g/node-value manifest :graphics-osx)))
+        (is (= :both (g/node-value manifest :graphics-web)))))
+    (testing "/app_manifest/android_support.appmanifest"
+      (let [manifest (test-util/resource-node project "/app_manifest/android_support.appmanifest")]
+        (is (= :legacy (g/node-value manifest :physics-2d)))
+        (is (= true (g/node-value manifest :physics-3d)))
+        (is (= false (g/node-value manifest :exclude-record)))
+        (is (= false (g/node-value manifest :exclude-profiler)))
+        (is (= false (g/node-value manifest :exclude-sound)))
+        (is (= false (g/node-value manifest :exclude-input)))
+        (is (= false (g/node-value manifest :exclude-liveupdate)))
+        (is (= false (g/node-value manifest :exclude-basis-transcoder)))
+        (is (= true (g/node-value manifest :use-android-support-lib)))
+        (is (= :open-gl (g/node-value manifest :graphics)))
+        (is (= :vulkan (g/node-value manifest :graphics-osx)))
+        (is (= :web-gl (g/node-value manifest :graphics-web)))))))
```

Comment:
Mostly indentation due to adding `(testing "..." ...)`  blocks, but also the tests reflect the split of the `:physics` property to `:physics-2d` and `:physics-3d`.